README.md

```diff
@@ -11,8 +11,8 @@ Before running, configure the test bot by adding an .env file with the following
 PREFIX=>
 TOKEN=[token]
 GUILD_ID=[guild id]
-VERIFIED_ROLE_NAME=[verified role name]
-ADMIN_ROLE_NAME=[admin role name]
+VERIFIED_ROLE_ID=[verified role name]
+ADMIN_ROLE_ID=[admin role name]
```

Comment:
Why was this changed to ID? Now you'd have to figure out the role number instead of just using the name.

Reply:
this is to prevent roles with the same name causing problems. a guild id is already needed, so you'll need to go into developer mode anyway. role IDs would not be that much harder to get